src/pages/editprofile.jsx

```diff
@@ -0,0 +1,149 @@
+                        className='border rounded-md w-full border-solid self-center border-light-gray/30 text-light-black px-[20px] py-[10px] '
+                    >
+                        <p className='flex justify-end'>+</p>
+                        <input type='file' hidden />
+                    </label>
+                </div>
+                <FormTitle title={t("Security")} />
+                <div className='flex flex-row justify-center my-5 gap-6'>
+                    <label className='w-4/12'>{t("Password")}</label>
+                    <Input type='name' name='password' />
+                </div>
+                <div className='flex flex-row justify-center my-5 gap-6'>
+                    <label className='w-4/12'>{t("confirmPassword")}</label>
+                    <Input type='password' name='password' />
+                </div>
+                <div className='flex gap-10 my-8'>
+                    <Button
+                        content={t("saveChanges")}
+                        filled='true'
+                        size='small'
+                        radius='md '
+                        shadow='shadow-lg'
+                    />
+                    <Button
+                        content={t("deleteAccount")}
+                        filled='true'
+                        size='small'
+                        radius='md '
+                        shadow='shadow-lg'
+                    />
+                    <Button
+                        content={t("Cancel")}
+                        filled='true'
+                        size='small'
+                        radius='md '
+                        shadow='shadow-lg'
+                    />
+                </div>
+
+                <FormTitle title={t("paymentMethods&Tickets")} />
+                <div className='flex gap-10 my-8'>
+                    <div className='flex flex-col gap-5'>
+                        <p>{t("3cardsAdded")}</p>
+                        <Button
+                            content={t("sHowCards")}
+                            filled='true'
+                            size='small'
+                            radius='md '
+                            shadow='shadow-lg'
+                        />
+                    </div>
+                    <div className='flex flex-col gap-5'>
+                        <p>{t("10ticketsRemaining")}</p>
+                        <Button
+                            content={t("buyTickets")}
+                            filled='true'
+                            size='small'
+                            radius='md '
+                            shadow='shadow-lg'
+                        />
+                    </div>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+export async function getStaticProps({ locale }) {
+    return {
+        props: {
+            ...(await serverSideTranslations(locale, ["profile", "common"])),
+        },
+    };
+}
```

Comment:
You didnt export withTranslation HOC to pass (t) object to the component for translation.